api/create.go

```diff
@@ -71,7 +72,7 @@ func CreateService(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 	s.Architecture = config.Cfg.Common.Architecture
-	s.DockerNetwork = fmt.Sprintf("%s_default", s.Db.Name) // following docker-compose naming format for compatibility
+	s.DockerNetwork = "spinup_services"
```

Comment:
Don't we need a unique network name for each cluster?

Reply:
Also, we need to change the network name for other services too. Eg. https://github.com/spinup-host/spinup/pull/114/files#diff-cc73ad2e2e501df6e7a96f7724d68dda0da92408743c43c55428303f5b67e2ffL183

Rather we can create the network name as a const variable across packages. thoughts?

Reply:
> Don't we need a unique network name for each cluster?
NM. Maybe we don't.

Reply:
you're right about the constant - will change that now.

Re unique network - I don't think we do either. Docker will generate unique IP addresses for each service so I don't see any problem yet.